server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMailboxesUpdateProcessor.java

```diff
@@ -128,6 +129,11 @@ private void validateParent(Mailbox mailbox, MailboxUpdateRequest updateRequest,
         }
     }
 
+    private boolean isParentIdInRequest(MailboxUpdateRequest updateRequest) {
+        return updateRequest.getParentId() != null
```

Comment:
We have to check different kind of input values for parentId:
- the parentId is not given -> Optional.empty
- the parentId is null (set this mailbox as a root) -> null
- the parentId is given -> Optional.of